gems/pending/spec/postgres_ha_admin/failover_databases_spec.rb

```diff
@@ -1,32 +1,48 @@
-describe FailoverDatabases do
+require 'postgres_ha_admin/failover_databases'
+require 'postgres_ha_admin/postgres_ha_admin'
+require 'spec/support/test_env_helper'
+require 'pg'
+
+describe PostgresHaAdmin::FailoverDatabases do
+  subject do
+    described_class.new(TestEnvHelper::SPEC_DIR, TestEnvHelper::SPEC_DIR, :dbname => 'travis', :user => 'travis')
```

Comment:
Why not use `CONFIG_DIR` instead of `SPEC_DIR` I think that is closer to what you will actually be doing right?